packages/office-ui-fabric-react/src/components/Panel/Panel.scss

```diff
@@ -92,6 +92,7 @@ $CommandBarHeight: 40px;
     @include right(auto);
     @include left(0);
     width: $Panel-width-xs;
+    @include drop-shadow(30px, 0px, 30px, -30px, .2);
```

Comment:
Use the [`ms-box-shadow()` mixin](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/sass/mixins/_Directionality.Mixins.scss#L275) here, and on line 71, for RTL support.